### PR TITLE
Clarify wording in General Settings and spelling/spacing fixes

### LINKED ISSUE #6344: Wording of options
Feedback on GUI, received via email. With respect to the general settings dialog, though the concept should apply to all dialogs.

> Wording can be improved for some options to make the purpose clearer and to shorten the text and apply some consistency.  Some suggestions follow:
> 
> Language (requires restart to fully take effect):  combo box  User default  collapsed 
> Do we mean restart of nvda or restart of windows? so wording could be changed to: 
> Language (changes take effect after restarting NVDA):  combo box  User default  collapsed
> 
> Save configuration on exit  check box  checked 
> could be re-worded to 
> Save configuration on exiting NVDA  check box  checked
> 
> Show exit options when exiting NVDA  check box  not checked 
> could be re-worded to 
> Confirm before exiting NVDA  check box  not checked
> 
> Automatically start NVDA after I log on to Windows  check box  not checked 
> could be re-worded to 
> Automatically start NVDA after logging on to Windows  check box  not checked
> 
> Use currently saved settings on the logon and other secure screens (requires administrator privileges)  button 
> could be shorter and re-worded to: 
> Use saved settings on logon and secure screens (requires administrator privileges)  button

### LINKED ISSUE #6741: NVDA menu and settings options either missing hotkeys or hotkey not visually underlined.
This issue initially only covered "Report formatting changes after the cursor".  For completeness, I have recorded below all NVDA menu and settings options which either do not have a hotkey or for which the hotkey is not underlined.

When you press NVDA+n and bring up the NVDA menu, the hotkey (the letter you can press to activate that item) is underlined on most options.  For instance, the "C" in "Configuration profiles" is underlined, indicating you can press C to directly open the configuration profiles screen.

In the NVDA menu and sub-menus, there are several options where the hotkey is not underlined in the menu item and should be.

In the main NVDA menu, "Tools" should have "T" underlined, and "Donate" should have "D" underlined.

In the Tools menu, ONLY "Manage add-ons" has a letter underlined ("a"), so the rest need their hotkey letter underlined.

In the Help menu, the "About" option should have the "A" underlined.

In the "Settings" dialog (looking at the new combined multi-settings dialog), the following options do not have hotkeys underlined:
- General Page: "Use NVDA on the Windows logon screen"
- Speech Page: "Automatic language switching",
	"Automatic dialect switching"
	"Trust Voice's language..." and
	"Capital pitch change percentage"
- Braille page: "Blind cursor"
	"Cursor blink rate" and 
	"Focus context presentation"
- Keyboard page: "Use Capslock as an NVDA modifier"
	"Use numpad Insert as an NVDA modifier"
	"Use extended Insert as an NVDA modifier"
	"Speech interrupt for typed characters"
	"Speech interrupt for Enter key" and
	"Beep if typing lowercase letters when caps lock is turned on"
- Review cursor: "Simple review mode"
- Object presentation: "Report background progress bars"
- Browse mode: "Automatic focus mode for focus changes"
	"Automatic focus mode for caret movement" and
	'Audio indication of focus and browse modes"
- Document formatting: "Cell borders" and
	"Report formatting changes after the cursor"
	
And for completeness:
- In the Add-ons manager, there is no hotkey displayed for the "Installed add-ons" list itself.
- In the Create portable copy dialog, the Browse button does not have a hotkey.

### PR DESCRIPTION
**Pull request by ThomasStivers**
### Link to issue number:
Fixes #6344, #6741 

### Summary of the issue:
* Fixed various capitalization and spelling errors  in docstrings and comments.
* Clarified wording on some controls in NVDA settings dialogs. (fixes #6344)
* Fixed accelerator conflict between output device combobox and audio ducking checkbox in synthesizer settings dialog.
* Added missing accelerators in document formatting, keyboard, object presentation, and review cursor settings dialogs. (Fixes #6741)
* Removed tabs from blank lines.

### Known issues with pull request:
There are still controls in some preferences dialogs without accelerators.

### Change log entry:
Added additional accelerator keys to preferences dialogs.

### CODE REVIEW

File: source/gui/settingsDialogs.py [hunk 1]
```diff
@@ -183,8 +183,8 @@ def makeSettings(self, settingsSizer):
 		else:
 			log.debugWarning("Could not set log level list to current log level")
 
-		# Translators: The label for a setting in general settings to allow NVDA to start after logging onto Windows (if checked, NvDA will start automatically after loggin into Windows; if not, user must start NVDA by pressing the shortcut key (CTRL+Alt+N by default).
-		self.startAfterLogonCheckBox = wx.CheckBox(self, label=_("&Automatically start NVDA after I log on to Windows"))
+		# Translators: The label for a setting in general settings to allow NVDA to start after logging onto Windows (if checked, NvDA will start automatically after logging into Windows; if not, user must start NVDA by pressing the shortcut key (CTRL+Alt+N by default).
```
Comment (derekriemer) [inline source/gui/settingsDialogs.py]: Please fix NVda while you're here.

**Review by derekriemer — COMMENTED** [review]
Everything looks fine in this PR. I recommend a pass from NV Access, barring any other UI issues they find. It might be good to have someone sighted look over the GUI changes on this PR though. @ThomasStivers if you have a chance before it's reviewed by NV Access, I did note one thing you could take a stab at fixing.

**Comment by derekriemer** [comment]
Congrats on your first PR

**Comment by feerrenrut** [comment]
@derekriemer There is some outstanding discussion of these changes on the issue. Have you looked over #6344 ?

**Comment by leonardder** [comment]
@ThomasStivers: Could you please fix the merge conflicts and merge latest master into this?

**Comment by ehollig** [comment]
@feerrenrut is there anything else that needs to be done on this PR?

**Comment by feerrenrut** [comment]
Since there has been no activity in a while, I'm going to take this on:
- there are conflicts to fix
- Some discussion on #6344 is not reflected here.

**Comment by feerrenrut** [comment]
@ThomasStivers Could you take a look at the diff / commit 91850e8 and let me know if you are happy with these changes? Then I can merge this.

**Review by feerrenrut — APPROVED** [review]

**Comment by josephsl** [comment]
Hi,

Regression found: NVDA settings/Object Presentation/Guess object position has wrong accelerator, caused by two ampersand signs (a PR to resolve this is on its way).

Thanks.

**Pull request merged**